src/core/base/source.d.ts

```diff
@@ -0,0 +1,3 @@
+export type UIPEditableSource = 'js' | 'html';
```

Comment:
d.ts is an output file that used with js sources, please use plain `source.ts` or `source.type.ts` file name